mmv1/third_party/terraform/.teamcity/components/builds/build_parameters.kt

```diff
@@ -211,7 +218,7 @@ fun BuildType.enableProjectSweep(){
     }
 }
 
-// ParametrizedWithType.terraformEnableProjectSweeper unsets an environment variable used to skip the sweeper for project resources
+// ParametrizedWithType.vcrEnvironmentVariables unsets an environment variable used to skip the sweeper for project resources
```

Comment:
It seems like maybe the function name in this comment was corrected, but not the rest of it?

Reply:
Thanks! I pushed a commit to address this